src/tests/functional/plugin/cpu/single_layer_tests/gather.cpp

```diff
@@ -1,345 +1,343 @@
+        ElementType,                       // Network precision
+        bool,                              // Is axis input constant
+        CPUSpecificParams,                 // CPU specific params
+        std::map<std::string, std::string> // Additional config
+> GatherLayerTestCPUParams;
+
+class GatherLayerTestCPU : public testing::WithParamInterface<GatherLayerTestCPUParams>,
+                           virtual public ov::test::SubgraphBaseTest, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<GatherLayerTestCPUParams> obj) {
+        std::vector<InputShape> inputShapes;
+        int64_t axis, batchDims;
+        ElementType netPrecision;
+        bool isAxisConstant;
+        CPUSpecificParams cpuParams;
+        std::map<std::string, std::string> additionalConfig;
+
+        std::tie(inputShapes, axis, batchDims, netPrecision, isAxisConstant, cpuParams, additionalConfig) = obj.param;
+
+        std::ostringstream result;
+        result << "IS=(";
+        for (size_t i = 0lu; i < inputShapes.size(); i++) {
+            result << CommonTestUtils::partialShape2str({inputShapes[i].first}) << (i < inputShapes.size() - 1lu ? "_" : "");
+        }
+        result << ")_TS=";
+        for (size_t i = 0lu; i < inputShapes.front().second.size(); i++) {
+            result << "{";
+            for (size_t j = 0lu; j < inputShapes.size(); j++) {
+                result << CommonTestUtils::vec2str(inputShapes[j].second[i]) << (j < inputShapes.size() - 1lu ? "_" : "");
+            }
+            result << "}_";
+        }
+        result << "axis=" << axis << "_";
+        result << "batchDims=" << batchDims << "_";
+        result << "netPrc=" << netPrecision << "_";
+        result << "constAx=" << (isAxisConstant ? "True" : "False") << "_";
+        result << CPUTestsBase::getTestCaseName(cpuParams);
+
+        if (!additionalConfig.empty()) {
+            result << "_PluginConf";
+            for (auto &item : additionalConfig) {
+                if (item.second == InferenceEngine::PluginConfigParams::YES)
+                    result << "_" << item.first << "=" << item.second;
+            }
+        }
+
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        std::vector<InputShape> inputShapes;
+        int64_t batchDims;
+        ElementType netPrecision;
+        bool isAxisConstant = true;
+        CPUSpecificParams cpuParams;
+        std::map<std::string, std::string> additionalConfig;
+        const ElementType intInputsPrecision = ElementType::i64;
+
+        std::tie(inputShapes, axis, batchDims, netPrecision, isAxisConstant, cpuParams, additionalConfig) = this->GetParam();
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        init_input_shapes(inputShapes);
+//        configuration.insert(additionalConfig.begin(), additionalConfig.end());
```

Comment:
It seems that this line should be uncommented since we need to provide the plugin config at least for BF16 precision testing.

Reply:
Done